classes/log/store.php

```diff
@@ -63,15 +64,59 @@ protected function is_event_ignored(event_base $event) {
         return $isdisabledevent;
     }
 
+    /**
+     * get eventid from logstore_standard_log
+     * @param event_base $event
+     * @return int
+     *
+     */
+    protected function get_event_id($event) {
+        global $DB;
+
+        $sqlparams = [];
+        $sqlparams['eventname'] = $event->eventname;
+        $sqlparams['component'] = $event->component;
+        $sqlparams['action'] = $event->action;
+        $sqlparams['target'] = $event->target;
+        $sqlparams['objecttable'] = $event->objecttable;
+        $sqlparams['objectid'] = $event->objectid;
+        $sqlparams['timecreated'] = $event->timecreated;
```

Comment:
is `timecreated` a reliable param to check on here?

i.e. can we be sure the timestamp of the event is the same as that of the logstore_standard_log?

Reply:
Yes the same data is passed to every logstore handler.